hpOneView/resources/resource.py

```diff
@@ -317,3 +317,8 @@ def get_utilization(self, id, fields=None, filter=None, refresh=False, view=None
         uri = "{0}/{1}/utilization{2}".format(self._uri, id, query)
 
         return self._connection.get(uri)
+
+    def __make_filter_query(self, filter):
```

Comment:
There is also a "query" query parameter so filter query differentiates
better I think.
On Jun 1, 2016 7:12 AM, "Camila Balestrin" <notifications@github.com> wrote:

> In hpOneView/resources/resource.py
> <https://github.com/HewlettPackard/python-hpOneView/pull/54#discussion_r65356543>
> :
>
> > @@ -317,3 +317,8 @@ def get_utilization(self, id, fields=None, filter=None, refresh=False, view=None
> >          uri = "{0}/{1}/utilization{2}".format(self._uri, id, query)
> >
> >          return self._connection.get(uri)
> > +
> > +    def __make_filter_query(self, filter):
>
> sounds better
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/HewlettPackard/python-hpOneView/pull/54/files/bce77488c5824a5598de0879a1acc6e8abcd4ae2#r65356543>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/ACRSxQZfkCxOd6TgRmDKdVqAAQWVMpLSks5qHYTvgaJpZM4Iq9O6>
> .
>